versioned_docs/version-2.6/pages-for-subheaders/install-upgrade-on-a-kubernetes-cluster.md

```diff
@@ -121,12 +121,6 @@ There are three recommended options for the source of the certificate used for T
 
 ### 4. Install cert-manager
 
-:::note
-
-New in v2.6.4, cert-manager versions 1.6.2 and 1.7.1 are compatible. We recommend v1.7.x because v 1.6.x will reach end-of-life on March 30, 2022.
-
-:::
-
```

Comment:
I think you should leave this in all places until 2.6 is EOL, which is September 2023.

Reply:
Updated in [4118b81](https://github.com/rancher/rancher-docs/pull/526/commits/4118b811ba8c58e7242be697be3da8ccec1361f9)